contextproject-ms3/src/main/java/contextproject/sorters/MaximumFlowPlaylistSorter.java

```diff
@@ -0,0 +1,14 @@
+package contextproject.sorters;
+
+import contextproject.models.Playlist;
+
+public class MaximumFlowPlaylistSorter implements PlaylistSorter{
+
+  @Override
+  public Playlist sort(Playlist playlist) {
+    Graph graph = new Graph(playlist);
+    MaxFlow mx = new MaxFlow(graph);
+    return (Playlist) mx.getOptimalPath();
```

Comment:
On my machine, this takes really long and then crashes. I have no idea why. Can we add logging to parts of this algorithm to at least show some progress?